docker-compose.yml

```diff
@@ -41,10 +43,29 @@ services:
       RAILS_ENV: development
     entrypoint: /home/app/students_crm/bin/start_dev.sh
 
+  pgadmin:
+    container_name: pgadmin
+    image: dpage/pgadmin4:6.12
+    depends_on:
+      - postgres
+    environment:
+      PGADMIN_DEFAULT_EMAIL: ${PGADMIN_DEFAULT_EMAIL}
```

Comment:
Since this is a development server, there is no need to store the `PGADMIN_DEFAULT_EMAIL` variable

Please commit the dummy email and password right in the file

Like `admin@example.com/password` or something

Reply:
On second thought, please remove the `pgadmin` block altogether.

1. pgAdmin is available to install locally
2. There are many other options besides that. So, every developer chooses what suits him the best
3. There is no need to start it for the development server to be able to work

Reply:
I am removing the block from the configuration so everyone can choose a convenient working method. 👍